sphinx/ext/napoleon/docstring.py

```diff
@@ -846,8 +848,13 @@ def combine_set(tokens):
 
 def _tokenize_type_spec(spec: str) -> List[str]:
     def postprocess(item):
-        if item.startswith("default"):
-            return [item[:7], item[7:]]
+        if item.startswith("default") and item != "default":
+            default = item[:7]
+            other = item[7:].lstrip()
+
+            whitespace = item[7:7 + len(item) - len(default) - len(other)]
+
+            return [default, whitespace, other]
```

Comment:
Can we use a single space character instead of `whitespace`? Is there any reason to pass the actual whitespace sequence?
```suggestion
            return [default, " ", other]
```

Reply:
see above, it is currently unclear what the official formats should be so we might have to rename `whitespace` to `delimiter`.

Reply:
I followed your suggestion for now